### DISCUSSION TITLE
Extract data from shapes in an excel table

### DISCUSSION THREAD

**Discussion by Cokoue** [jgraph/drawio-desktop#1389] [Q&A] [answered]
Hello,

Recently switched from visio to drawio, I used to extract data from logigrams in Visio to excel. This data is used for other applications such as tables.
Do you know if drawio supports this type of extraction and how?

Thank you !

**Comment by davidjgraph** [comment, marked as answer]
I've never heard the word "logigrams", so I would assume no, it's not possible.

**Reply by Cokoue** [reply]
Sorry, this is a very bad translation from my old French (thought it was the same word). The correct word is diagram.

**Reply by davidjgraph** [reply]
Pas grave. There is no current export to CSV, though it has been requested https://github.com/jgraph/drawio/issues/3729
